sklearn/linear_model/_base.py

```diff
@@ -636,6 +638,14 @@ class LinearRegression(MultiOutputMixin, RegressorMixin, LinearModel):
     array([16.])
     """
 
+    _parameter_constraints = {
+        "fit_intercept": [bool],
+        "normalize": [StrOptions({"deprecated"}), bool],
```

Comment:
waiting for https://github.com/scikit-learn/scikit-learn/pull/23459 before merging this PR such that we can set `StrOptions({"deprecated"}, internal={"deprecated"})` to not show this option in the error message

Reply:
yes please